core/designsystem/src/main/kotlin/com/google/samples/apps/nowinandroid/core/designsystem/component/scrollbar/LazyScrollbarUtilities.kt

```diff
@@ -33,7 +33,7 @@ import kotlin.math.abs
  * @return a [Float] in the range [firstItemPosition..nextItemPosition) where nextItemPosition
  * is the index of the consecutive item along the major axis.
  * */
-internal inline fun <LazyState : ScrollableState, LazyStateItem> LazyState.interpolateFirstItemIndex(
+internal inline fun <LazyState : ScrollableState, LazyStateItem> LazyState.interpolateIndex(
```

Comment:
`interpolateFirstVisibleItemIndex` may be a better name here as the index returned is of the first visible item in the grid